azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -1091,20 +1098,88 @@ public int getExecutionId() {
     return this.execId;
   }
 
+  public Set<JobRunner> getActiveJobRunners() {
+    return ImmutableSet.copyOf(this.activeJobRunners);
+  }
+
+  // Class helps report the flow start and stop events.
+  private class FlowRunnerEventListener implements EventListener {
+
+    public FlowRunnerEventListener() {
+    }
+
+    private synchronized Map<String, String> getFlowMetadata(final FlowRunner flowRunner) {
+      final ExecutableFlow flow = flowRunner.getExecutableFlow();
+      final Map<String, String> metaData = new HashMap<>();
+      metaData.put("flowName", flow.getId());
+      metaData.put("azkabanHost",
+          flowRunner.azkabanProps.getString("azkaban.server.hostname", "unknown"));
```

Comment:
Use constant azkaban.Constants.ConfigurationKeys#AZKABAN_SERVER_HOST_NAME

Reply:
Ok, will use that.

Reply:
done.